mosaic-runtime/src/commonMain/kotlin/com/jakewharton/mosaic/mosaic.kt

```diff
@@ -170,7 +170,7 @@ internal class MosaicComposition(
 	private val composeContext: CoroutineContext = coroutineScope.coroutineContext + job + clock
 	val scope = CoroutineScope(composeContext)
 
-	private val applier = MosaicNodeApplier(::performLayout)
```

Comment:
This behavior was flat out wrong. I also made the same mistake in Redwood.

If you use `movableContentOf` and move something within a single recomposition (such as from within `Row { }` to a `Column { }`), you'll actually get two pairs of calls to begin/end where the former performs the node removal and the latter performs the node add. If you render solely on the after callback, you'll render an invalid state which is missing all of the content from `movableContentOf`s.